### DISCUSSION TITLE
CTFd connection

### DISCUSSION THREAD

**Discussion by axxic3** [juice-shop/multi-juicer#235] [Q&A] [answered]
Hello, I am planning on hosting a ctf event for a local group. Currently, I have a CTFd server hosted in digitalocean. With the need for hosting multiple juice-shop users, we are leaning towards using multi-juicer. My question is, how would I connect those to the CTFd server to keep track of team scores in a central location?

Thanks!

**Comment by J12934** [comment, marked as answer]
Hi @axxic3 

MultiJuicer itself has no integration with CTFd.

You can still use it, by setting up configuring JuiceShop to run in CTF mode to output flags and import them into CTDd. See https://pwning.owasp-juice.shop/companion-guide/latest/part4/ctf.html

For that you'd need to configure the NODE_ENV for JuiceShop to be CTF (https://github.com/juice-shop/multi-juicer/blob/main/helm/multi-juicer/values.yaml#L118) or set the related flag in the MultiJuicer managed config file (https://github.com/juice-shop/multi-juicer/blob/main/helm/multi-juicer/values.yaml#L107)

Participants will still need to copy out the flag values from it JuiceShop into CTFd, there is no automatic syncing setup between MJ and CTFd.

You can also try the experimantal and very simpel MJ scoreboard. That's eneabled for every MJ installation just "hidden" under the unlinked `/balancer/score-board` url path.

**Reply by axxic3** [reply]
Excellent, copying flag values shouldn't be a problem I just wanted to know if it was possible for us to use flags with CTFd before digging too deep with JuiceShop. Thanks for the response!
